lib/grape/locale/en.yml

```diff
@@ -10,6 +10,9 @@ en:
         values: 'does not have a valid value'
         except_values: 'has a value not allowed'
         same_as: 'is not the same as %{parameter}'
+        length_mismatch_both: 'is expected to have a length within %{min_length} and %{max_length}'
```

Comment:
let's match the variables of the other errors

```
length
length_min
length_max
```

Reply:
done 👍